server.js

```diff
@@ -0,0 +1,59 @@
+const http = require('http');
+const fs = require('fs');
+//const public = require('./public')
+
+const PORT = process.env.PORT || 3000;
+
+const server = http.createServer((request, response) => {
+  const uri = request.url;
+  const reqMeth = request.method;
+console.log(uri)
+  function reqResp(uri, type) {
```

Comment:
If you parse the `request.uri` you can parse out the file type, you do not need to hard code it.